src/content/administration/dashboard.mdx

```diff
@@ -4,75 +4,125 @@ description: Admin Dashboard
 id: dashboard
 ---
 
+import Image from '@theme/Image';
 import TiersList from '@theme/TiersList';
 
 # Admin Dashboard  <TiersList tiers="Scale Enterprise Self-Hosted" />
 
-The Admin Dashboard provides a web UI for administering some features of Okteto.
+The Admin Dashboard provides a web UI to see various details of your Okteto instance in a centralized location and offers features for managing parts of the Okteto experience for your users.
 
 You can access the Admin Dashboard by clicking on the `Admin` icon on the left sidebar.
 
+<p align="center"><Image src="/docs/administration/images/admin_dashboard_tabs.png" alt="Admin dashboard tabs" width="900" /></p>
+
+This Admin view consists of the following views currently:
+
+- [General](#overview)
+- [Users](#users)
+- [Nodes](#nodes)
+- [Namespaces](#namespaces)
+- [Previews](#previews)
+- [Secrets](#secrets)
+- [Catalog](#catalog)
+- [Installation](#installation)
+
+We will cover these in more detail below.
+
 ## Accessing the dashboard
 
 Only accounts with the administrator role will be able to access the Administration Dashboard. The first user signing up into your Okteto instance will automatically become the initial administrator.
 
 You can promote other accounts to the administrator role in the `Users` section of the Admin Dashboard or by adding the `dev.okteto.com/super: "true"` label to the corresponding `serviceAccount` in Kubernetes.
 
-## Overview Section
+## Overview
 
-The overview section of the Admin Dashboard shows you the following information:
+The overview section of the Admin Dashboard is designed to give you a high level view of your entire Okteto instance. From here, you can drill down into more specific settings and details. This section currently shows you the following information:
 
+- Your current pricing plan
+- Okteto Helm chart version
+- Okteto CLI version
 - Expiration date of your license
 - Number of available seats
 - Number of nodes in your cluster
-- Number of namespaces managed by Okteto
+- Number of namespaces in your Okteto instance
 - Number of preview environments currently active
+- Git SSH key
 - Autoscaler configuration
 - Garbage Collector configuration
 
-## Users Section
+With the exception of the `Settings` section, this data is informative only. The `Settings` section contains a few configurable settings to help you customize your Okteto instance as necessary.
+
+## Users
+
+From this section you can manage all users in your Okteto instance. This is a list of all users that have created an account in your Okteto instance, their respective roles, and with appropriate actions to manage their membership.
+
+:::info
+
+The `User` column is also the user's `namespace` name/id. For example, a `User` value of "darryl" means that that user's namespace will also be called "darryl".
```

Comment:
This is only true for the default namespace. A user might create other namespaces with random names

Reply:
@pchico83 Good catch. If a user creates 3 additional namespaces, those will only show up in the `Namespace` tab and _not_ the `Users` tab, correct?

Reply:
Right. The user tab only shows the user, it doesn't show information about the default namespace either